scripts/globals/garrison_data.lua

```diff
@@ -0,0 +1,622 @@
+            "0x0100080669106B206B306B406B50FE6000700000"
+        },
+    },
+    --level 75 garrison looks
+    [75] =
+    {
+        [xi.nation.SANDORIA] =
+        {
+            -- (Missing capture)
+        },
+
+        [xi.nation.BASTOK] =
+        {
+            "0x010002071C1070201C301C401C50C46000700000",
+        },
+
+        [xi.nation.WINDURST] =
+        {
+            -- (Missing capture)
+        },
+    },
+}
+
+-- Loot is determined by LevelCap
+xi.garrison.loot =
+{
+    --level 20 garrison loot
+    [20] =
+    {
+        { itemid = xi.items.DRAGON_CHRONICLES, droprate = 1000 },
+        { itemid = xi.items.GARRISON_TUNICA,   droprate =  350 },
+        { itemid = xi.items.GARRISON_BOOTS,    droprate =  350 },
+        { itemid = xi.items.GARRISON_HOSE,     droprate =  350 },
+        { itemid = xi.items.GARRISON_GLOVES,   droprate =  350 },
+        { itemid = xi.items.GARRISON_SALLET,   droprate =  350 },
+    },
+    --level 30 garrison loot
+    [30] =
+    {
+        { itemid = xi.items.DRAGON_CHRONICLES, droprate = 1000 },
+        { itemid = xi.items.MILITARY_GUN,      droprate =  350 },
+        { itemid = xi.items.MILITARY_POLE,     droprate =  350 },
+        { itemid = xi.items.MILITARY_HARP,     droprate =  350 },
+        { itemid = xi.items.MILITARY_PICK,     droprate =  350 },
+        { itemid = xi.items.MILITARY_SPEAR,    droprate =  350 },
+        { itemid = xi.items.MILITARY_AXE,      droprate =  350 },
+    },
+    --level 40 garrison loot
+    [40] =
+    {
+        { itemid = xi.items.DRAGON_CHRONICLES, droprate = 1000 },
+        { itemid = xi.items.VARIABLE_MANTLE,   droprate =  350 },
+        { itemid = xi.items.VARIABLE_CAPE,     droprate =  350 },
+        { itemid = xi.items.PROTEAN_RING,      droprate =  350 },
+        { itemid = xi.items.VARIABLE_RING,     droprate =  350 },
+        { itemid = xi.items.MECURIAL_EARRING,  droprate =  350 },
+    },
+    --level 50 garrison loot
+    [50] =
+    {
+        { itemid = xi.items.DRAGON_CHRONICLES, droprate = 1000 },
+        { itemid = xi.items.UNDEAD_EARRING,    droprate =  350 },
+        { itemid = xi.items.ARCANA_EARRING,    droprate =  350 },
+        { itemid = xi.items.VERMIN_EARRING,    droprate =  350 },
+        { itemid = xi.items.BIRD_EARRING,      droprate =  350 },
+        { itemid = xi.items.AMORPH_EARRING,    droprate =  350 },
+        { itemid = xi.items.LIZARD_EARRING,    droprate =  350 },
+        { itemid = xi.items.AQUAN_EARRING,     droprate =  350 },
+        { itemid = xi.items.PLANTOID_EARRING,  droprate =  350 },
+        { itemid = xi.items.BEAST_EARRING,     droprate =  350 },
+        { itemid = xi.items.DEMON_EARRING,     droprate =  350 },
+        { itemid = xi.items.DRAGON_EARRING,    droprate =  350 },
+        { itemid = xi.items.REFRESH_EARRING,   droprate =  350 },
+        { itemid = xi.items.ACCURATE_EARRING,  droprate =  350 },
+    },
+    --level 75 garrison loot
+    --TODO: 75 era module
```

Comment:
Whats left TODO here?

Reply:
Multiple sources confirmed Cape Terrigan / Vollbow should be "uncapped" (99).
Changed all `75` references to `99`, and now defer to `main.MAX_LEVEL` in server settings.

Reply:
uncapped and "capped at max level" are 2 diff things, be sure use corrected terminology to avoid confusion. Capped at max level will strip buffs, uncapped won't. (I expect you have the right thing and just had the wrong wording)